packages/shared/src/constants.ts

```diff
@@ -6,8 +6,8 @@ export enum Difficulty {
 }
 
 export enum Marker {
-  o = 'o',
-  x = 'x',
+  o = 'O',
```

Comment:
Purely so the letters would be capitalized, alternatively could have transformed through CSS.